app/design/frontend/Gorban/Sergey/web/css/source/_typography.less

```diff
@@ -0,0 +1,8 @@
+& when (@media-common = true) {
```

Comment:
Such file already exists - https://github.com/magento/magento2/blob/2.2.6/app/design/frontend/Magento/blank/web/css/source/_typography.less
Do now overwrite the content of the existing files. You must preserve existing content or use another file like 
https://github.com/magento/magento2/blob/2.2.6/lib/web/css/source/_extend.less